deploy/redeploy_radiation.sh

```
@@ -0,0 +1,99 @@
+            SAVEECOBOT_API_KEY="$2"
+            shift 2
+            ;;
+        -s|--sensors-url)
+            SAVEECOBOT_SENSORS_URL="$2"
+            shift 2
+            ;;
+        -d|--data-url)
+            SAVEECOBOT_DATA_URL="$2"
+            shift 2
+            ;;
+        -up|--data-url)
```

Shellcheck (0.10.0)

[warning] This pattern always overrides a later one on line 36.

(SC2221)